### ISSUE TITLE
Dog page

### ISSUE THREAD

**Issue by Wgatson23** [jstnknr/com6338-10-10-Folse-Gatson-Keiner#1]
Began the creation of the dog page.

**Comment by Wgatson23** [comment]
I created and added code for HTML and CSS. 
Pulled, added HTML, and addition CSS-
Today I'll edit my temporary menu and link other pages.

**Comment by rachelfolse** [comment]
Hello William,

I saw what you did with the dog page. I like the facts and everything you put. However, I think the navigation and footer needs to be consistent across the board for the whole website. Also, for the photos, I think that is where we should use an API and JavaScript. I noticed you put the HTML on your JavaScript page. Let me know if you need help with this.

**Comment by Wgatson23** [comment]
Agreed! I was wondering about that this morning. I’ve made the changes to the header and footer.  As far as the API, I’m still trying to figure that out. Any assistance will be appreciated.

On Apr 9, 2024, at 1:52 PM, rachelfolse ***@***.***> wrote:

[External Email]


Hello William,

I saw what you did with the dog page. I like the facts and everything you put. However, I think the navigation and footer needs to be consistent across the board for the whole website. Also, for the photos, I think that is where we should use an API and JavaScript. I noticed you put the HTML on your JavaScript page. Let me know if you need help with this.

**Comment by rachelfolse** [comment]
Hello! I think I pretty much figured out the API for the dog page. It took me a reallyyy long time but i wanted to get this assignment done as soon as possible. Take a look at the code and you should basically be able to do something similar for the cat page. It's basically the same API but instead of get a dog it's get a cat. Let me know if you need any further assistance. Thank you!

**Comment by Wgatson23** [comment]
Thank you! I really like the additions you made. Let me know if it’s anything else the group needs to get this assignment complete.

On Apr 9, 2024, at 10:21 PM, rachelfolse ***@***.***> wrote:

[External Email]


Hello! I think I pretty much figured out the API for the dog page. It took me a reallyyy long time but i wanted to get this assignment done as soon as possible. Take a look at the code and you should basically be able to do something similar for the cat page. It's basically the same API but instead of get a dog it's get a cat. Let me know if you need any further assistance. Thank you!
